api/tests/devcomp/acceptance/scripts/get-proposals_test.js

```diff
@@ -0,0 +1,395 @@
+import { getElements } from '../../../../scripts/modulix/get-elements-csv.js';
+import { getProposals, getProposalsListAsCsv } from '../../../../scripts/modulix/get-proposals-csv.js';
+import { expect } from '../../../test-helper.js';
+
+describe('Acceptance | Script | Get Proposals as CSV', function () {
+  const modulesListAsJs = [
```

Comment:
si tu utilises le même module dans les 3 tests, on pourrait pas sortir cette donnée dans un fichier à part et l'importer pour les 3 tests ?

Reply:
Hmmm why not. D'autres avis ?

Reply:
Discuté ensemble, on préfère aller dans le sens de créer un `builder` de contenus de Modules.